backend/src/routes/userRoute.js

```diff
@@ -11,4 +11,5 @@ router.post(
   AuthController.authenticateToken,
   UserController.changePassword
 );
+router.patch("/pfp", AuthController.authenticateToken, UserController.changePFP);
```

Comment:
- Once you add the `multer` and `upload` variables, I addressed above. 
- Please change your route to the following:
```suggestion
router.patch(
  "/pfp",
  AuthController.authenticateToken,
  upload.single("profileurl"),
  UserController.changePFP
);
```